crates/cast/tests/cli/main.rs

```diff
@@ -1482,6 +1482,41 @@ casttest!(event_decode, |_prj, cmd| {
 "#]]);
 });
 
+casttest!(error_decode_with_sig, |_prj, cmd| {
+    cmd.args(["decode-error", "--sig", "AnotherValueTooHigh(uint256,address)", "0x7191bc6200000000000000000000000000000000000000000000000000000000000000650000000000000000000000000000000000000000000000000000000000D0004F"]).assert_success().stdout_eq(str![[r#"
+101
+0x0000000000000000000000000000000000D0004F
+
+"#]]);
+});
```

Comment:
Would you mind adding an additional case for the `--json` output here?

Reply:
added in https://github.com/foundry-rs/foundry/pull/9428/commits/d349c9a3cbf22e65db5670ed3d047712926642ea